src/Script/Composer/README.md

```diff
@@ -0,0 +1,39 @@
+## Removing Unused Services
+**NOTE:** This feature is currently in beta. If you have general questions about usage or would like to report a 
+bug, please open an issue with us [here](https://github.com/googleapis/google-api-php-client/issues/new/choose). If 
+you have feedback on the implementation, please visit the [open discussion](https://github.com/aws/aws-sdk-php/discussions/2420) 
+we have on the topic.
+
+To avoid shipping unused services, specify which services you would like to use in your `composer.json` file and
+run the `Aws\\Script\\Composer::removeUnusedServices` script.   
```

Comment:
I thought you mentioned this would be run automatically on composer install; how does that happen?  Just the scripts tag below?